src/terraform-provider-signalform/signalform/util_test.go

```diff
@@ -45,7 +45,7 @@ func TestSendRequestFail(t *testing.T) {
 func TestSendRequestTimeout(t *testing.T) {
 	timeout := time.Duration(1 * time.Second)
 	server := httptest.NewServer(http.TimeoutHandler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		time.Sleep(timeout)
+		time.Sleep(2 * timeout)
```

Comment:
Sorry, what's wrong with the 1 second timeout?

Reply:
I explained in the commit message; the short version is: the timeouthandler and the handlerfunc are sleeping for the same amount of time, so they're racing. If the handlerfunc finishes sleeping before the timeouthandler, then it'll return 200 and the test will fail. I can reproduce this failure consistently on my workstation but I think it's just flaky in general. The solution is to make the handlerfunc sleep long enough that the race is no longer an issue.

Reply:
By the way, while we're discussing this, I would actually suggest deleting this test entirely. It's not testing any of the functionality of signalform itself; it's really only verifying that http.TimeoutHandler works the way the standard library says it should.